app/views/home/index.html.erb

```diff
@@ -1,2 +1,7 @@
-<h1>Home#index</h1>
-<p>Find me in app/views/home/index.html.erb</p>
+<%@link.in_groups_of(3).each do%>
+<div class="d-flex flex-row justify-content-evenly">
+<%=render partial: "partials/vedio-card", locals: {link: Link.first}%>
+<%=render partial: "partials/vedio-card", locals: {link: Link.first}%>
+<%=render partial: "partials/vedio-card", locals: {link: Link.first}%>
+</div>
+<%end%>
```

Comment:
can you please explain this code what exactly it is doing? and any specific reason why we rendering same partial 3 times? and also can you fix the indentation for this file.

Reply:
In order to show three videos in each column i called them in groups of three
however on second thought i think it is possible to use another '.each do' block inside this to avoid writing the code thrice
i will do that instead